src/main/java/se/sundsvall/supportmanagement/integration/db/model/NotificationEntity.java

```diff
@@ -68,6 +71,9 @@ public class NotificationEntity {
 	@Column(name = "errand_id")
 	private String errandId;
 
+	@Column(name = "errand_number")
+	private String errandNumber;
```

Comment:
Do we need this when we have errandId?

errandId kan be mapped to an ErrandEntity in this entity (manyToOne). From the errandEntity, you can get errandNumber.

If you don't want to make this mapping in the entity, you can fetch it with: `ErrandRepository.findById().getErrandNumber()`